src/iarc7_motion/task_command_handler.py

```diff
@@ -250,23 +264,15 @@ def handle_ground_interaction_done(self, status, result):
             rospy.logerr('Ground interaction done callback received with no task callback available')
 
     """
-    Sends twist (x, y, z, and angular velocities) to LLM
+    Sends motion point stamped array to LLM
 
     Args:
-        twist: TwistStampedArray (array of twists) or just a single twist stamped.
+        motion_point_stamped_array: MotionPointStampedArray
     """
-    def _publish_twist(self, twist):
-        if type(twist) is TwistStampedArray:
-            self._last_twist = twist.twists[-1]
```

Comment:
Let's rip out `self._last_twist` entirely if we don't use it anymore

Reply:
And rip it out of the state monitor entirely

Reply:
Agreed

Reply:
Can't be ripped out but added a fix that is in the spirit of the way it used to be.

Reply:
Some more was removed as discussed.